evtx/Maps/System_6008.map

```diff
@@ -0,0 +1,43 @@
+Author: Hyun Yi @hyuunnn
+Description: Unexpected system shutdown
+EventId: 6008
+Channel: "System"
+Maps:
+  -
+    Property: PayloadData1
+    PropertyValue: "Time, Date : %Data%"
+    Values:
+      -
+        Name: Data
+        Value: "/Event/EventData/Data"
```

Comment:
Did you test this? Did it grab data? It looks different than most other maps on this line.

Reply:
![image](https://user-images.githubusercontent.com/15611739/101279820-4432b600-3808-11eb-9930-3485d5612e75.png)

It works correctly.